doc/architectural/front-page.md

```diff
@@ -43,13 +41,16 @@ Contributing to the code base
 2. Clone the repository `git clone git@github.com:YOURNAME/cbmc.git`
 3. Create a branch from the `develop` branch (default branch)
```

Comment:
Points 1 and 2 should probably both be updated a touch too... I'd suggest something like:
1. For the CBMC repository on github
2. Clone your fork with `git clone......`